FLAGWORD_ASSIGNMENTS.s

```diff
@@ -365,7 +365,7 @@ TRACKBIT	=	BIT5
 		
 # BIT 3 FLAG 1	(S)
 SLOPESW		=	027D			#	ITERATE WITH BIAS	ITERATE WITH REGULAR
-						#	METHOD IN ITERATOR	FALSI METHOD IN
+						#	METHOD IN ITERATOR	FALSE METHOD IN
```

Comment:
`FALSI` is used in [source](http://www.ibiblio.org/apollo/ScansForConversion/Luminary099/0068.jpg)